### PR TITLE
Add documentation for Card

### PR DESCRIPTION
**Pull request by joannaquu**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Added documentation for `Card`

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1474)

### CODE REVIEW

File: ios/docs/Controls/Card.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Card
+
+## Overview
+The `Card` component is a framework for organizing content within the confines of a container. It's main function is to provide the scaffolding for hosting actions and content for a single topic within a card.
```
Comment (lyzhan7) [inline ios/docs/Controls/Card.md:4]: Its* main function

**Review by lyzhan7 — APPROVED** [review]

File: ios/docs/Controls/Card.md (hunk 1 above)
Comment (joannaquu) [reply ios/docs/Controls/Card.md:4]: Nice catch! I got this from https://react.fluentui.dev/?path=/docs/preview-components-card-card--default. Do you happen to know who owns this page?

**Comment by lyzhan7** [comment]
I've seen several PRs now for adding documentation to the FUA controls, was wondering what the context was behind adding these? Is this all brand new documentation, or are we moving the documentation here from somewhere else?

**Comment by joannaquu** [comment]
> I've seen several PRs now for adding documentation to the FUA controls, was wondering what the context was behind adding these? Is this all brand new documentation, or are we moving the documentation here from somewhere else?

This was just a side task that Hunter recommended I work on while I was waiting for others to return. It's brand new documentation.

**Pull request closed without merging**